README.md

```diff
@@ -5966,9 +6015,10 @@ The table and chart below show the performance results.
 <a id="markdown-known-issues" name="known-issues"></a>
 Fixes for the following issues will be released shortly:
 * The inference hyperparameter search is not available in this release for T5 and mT5.
-* Accuracy and performance measurement for GPT-3 is currently not supported. Please use the NeMo Megatron 22.05 inference container to use this feature.
+* Accuracy and performance measurement for GPT is currently not supported. Please use the NeMo Megatron 22.05 inference container to use this feature.
 * The fine-tuning SQuAD results for T5 are lower than expected.
-* There has been a slight regression in T5 performance and this will be addressed in an upcoming release.
 * Evaluation for GPT has been tested for PP <=2 and may have issues for PP >2. It is recommended to convert to TP only for Evaluation.
 * Transformer Engine (TE)-based GPT models are currently not supported for any Parameter Efficient Fine Tuning (PEFT) techniques - this will be added soon.
 * TE-based GPT Eval will take more memory than non-TE-based GPT Eval.
+* Iteration per second metric is incorrectly displayed on the Logs progress bar - this will be addressed in the next release. Instead, please use train step timing in Weights & Biases or TensorBoard.
+* Please note that Batch Size Ramp-up Support for GPT is currently working only with FusedAdam optimizer.
```

Comment:
Can we add one more known issue?

"Currently ramp-up batch size only works with optimizer fused_adam"